texttospeech/Gemfile.lock

```diff
@@ -70,8 +52,8 @@ PLATFORMS
 
 DEPENDENCIES
   google-cloud-text_to_speech
-  rack-test
-  rspec
+  minitest (~> 5.14)
+  rake (>= 12.0)
 
 BUNDLED WITH
-   1.16.2
+   2.1.4
```

Comment:
this will probably upset kokoro. can probably be manually changed to 1.17.3 to pass